pkg/tsdb/elasticsearch/client/index_pattern.go

```diff
@@ -71,9 +71,10 @@ var datePatternReplacements = map[string]string{
 
 type IndexPattern interface {
 	GetIndices(timeRange backend.TimeRange) ([]string, error)
+	GetIndexForToday(interval string, pattern string) ([]string, error)
```

Comment:
Instead of adding `GetIndexForToday` to the interface, can you create a `backend.TimeRange` and call `GetIndices` instead in the `validateIndex` function?

Reply:
Good point, I replaced it with GetIndices with range for now()